saw-core/src/Verifier/SAW/FiniteValue.hs

```diff
@@ -309,27 +309,41 @@ scFirstOrderValue sc fv =
 
 -- Parsing values from lists of booleans ---------------------------------------
 
-readFiniteValue' :: FiniteType -> S.StateT [Bool] Maybe FiniteValue
-readFiniteValue' ft =
+data Endianness
```

Comment:
I feel sure we've defined this datatype elsewhere before.  Can we grab it from some upstream dependency instead?

Reply:
Heh. I figured the same thing. Checking right now, I can find three right away, one in `crucible-server` and two in `macaw`. It would be strange to have `saw-core` depend on either. I could imagine it might make sense to have this be part of `bv-sized`, and to adapt `saw-core` to use that library. But that's a bigger refactoring than I think belongs in this PR. 😛

Reply:
Fair enough.